<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>B8</class>
 <widget class="QWidget" name="B8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(128, 128, 128, 128);</string>
  </property>
  <widget class="QLabel" name="lbIMGFondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/Resources/PNG/BackGround.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIMPPant">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>633</y>
     <width>134</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 208, 200);</string>
   </property>
   <property name="text">
    <string>IMP PANT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUnit3">
   <property name="geometry">
    <rect>
     <x>835</x>
     <y>649</y>
     <width>190</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(175, 175, 175);</string>
   </property>
   <property name="text">
    <string>    UNIT3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_0">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>338</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>DEMANDA DEL QUEMADOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_1">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>354</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MODO QUEMADOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_2">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>370</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>B (PILOTO)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbValor_0">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>65</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 11</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>118</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 9</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_6">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>219</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>ENCENDEDOR 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 8</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>445</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 7</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_9">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>495</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>ENCENDEDOR 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_10">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>440</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_11">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>365</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_12">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_13">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>115</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_14">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>65</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_15">
   <property name="geometry">
    <rect>
     <x>905</x>
     <y>575</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>VÁL PRE 6</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_16">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>625</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>ENCENDIDO MANUAL HABILITADO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHabDes">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>619</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 208, 200);</string>
   </property>
   <property name="text">
    <string>HAB/DESHAB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_17">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>535</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MÚLTIPLE A (EXTERNO)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_18">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>560</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MÚLTIPLE B (PILOTO)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_19">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>585</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(151, 0, 226);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MÚLTIPLE C (INTERNO)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTexto_20">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>608</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78,46, 28);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>SUMINISTRO DE GAS AUXILIAR</string>
   </property>
  </widget>
  <widget class="Flama_SectorA" name="flama_SectorA">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorB" name="flama_SectorB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorC" name="flama_SectorC">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorD" name="flama_SectorD">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorE" name="flama_SectorE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorF" name="flama_SectorF">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorG" name="flama_SectorG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorH" name="flama_SectorH">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorI" name="flama_SectorI">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorJ" name="flama_SectorJ">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorK" name="flama_SectorK">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <widget class="Flama_SectorL" name="flama_SectorL">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>662</height>
    </rect>
   </property>
  </widget>
  <zorder>lbIMGFondo</zorder>
  <zorder>lbUnit3</zorder>
  <zorder>lbTexto_0</zorder>
  <zorder>lbTexto_1</zorder>
  <zorder>lbTexto_2</zorder>
  <zorder>lbValor_0</zorder>
  <zorder>lbTexto_3</zorder>
  <zorder>lbTexto_4</zorder>
  <zorder>lbTexto_5</zorder>
  <zorder>lbTexto_6</zorder>
  <zorder>lbTexto_7</zorder>
  <zorder>lbTexto_8</zorder>
  <zorder>lbTexto_9</zorder>
  <zorder>lbTexto_10</zorder>
  <zorder>lbTexto_11</zorder>
  <zorder>lbTexto_12</zorder>
  <zorder>lbTexto_13</zorder>
  <zorder>lbTexto_14</zorder>
  <zorder>lbTexto_15</zorder>
  <zorder>lbTexto_16</zorder>
  <zorder>lbTexto_17</zorder>
  <zorder>lbTexto_18</zorder>
  <zorder>lbTexto_19</zorder>
  <zorder>lbTexto_20</zorder>
  <zorder>flama_SectorA</zorder>
  <zorder>flama_SectorB</zorder>
  <zorder>flama_SectorC</zorder>
  <zorder>flama_SectorD</zorder>
  <zorder>flama_SectorE</zorder>
  <zorder>flama_SectorF</zorder>
  <zorder>flama_SectorG</zorder>
  <zorder>flama_SectorH</zorder>
  <zorder>flama_SectorI</zorder>
  <zorder>flama_SectorJ</zorder>
  <zorder>flama_SectorK</zorder>
  <zorder>flama_SectorL</zorder>
  <zorder>btnHabDes</zorder>
  <zorder>btnIMPPant</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Flama_SectorA</class>
   <extends>QWidget</extends>
   <header>flama_sectora.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorB</class>
   <extends>QWidget</extends>
   <header>flama_sectorb.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorC</class>
   <extends>QWidget</extends>
   <header>flama_sectorc.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorD</class>
   <extends>QWidget</extends>
   <header>flama_sectord.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorE</class>
   <extends>QWidget</extends>
   <header>flama_sectore.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorF</class>
   <extends>QWidget</extends>
   <header>flama_sectorf.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorG</class>
   <extends>QWidget</extends>
   <header>flama_sectorg.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorH</class>
   <extends>QWidget</extends>
   <header>flama_sectorh.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorI</class>
   <extends>QWidget</extends>
   <header>flama_sectori.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorJ</class>
   <extends>QWidget</extends>
   <header>flama_sectorj.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorK</class>
   <extends>QWidget</extends>
   <header>flama_sectork.h</header>
  </customwidget>
  <customwidget>
   <class>Flama_SectorL</class>
   <extends>QWidget</extends>
   <header>flama_sectorl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
